Lint warning is issued in build of  pkg/storage

Golint currently generates a warning during a build (e.g. make test):

pkg/storage/storage.go:28:7:warning: exported const NoReleasesErr should have comment or be unexported (golint)

--- a/pkg/storage/storage.go
+++ b/pkg/storage/storage.go
@@ -25,6 +25,7 @@ import (
 	"k8s.io/helm/pkg/storage/driver"
 )
 
+// NoReleasesErr indicates that a given release cannot be found
 const NoReleasesErr = "has no deployed releases"
 
 // Storage represents a storage engine for a Release.

